platforms/Windows/sdk/drd/sdk.wxs

```diff
@@ -1,19 +1,16 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <Wix xmlns="http://wixtoolset.org/schemas/v4/wxs">
 
-  <?if $(ProductArchitecture) = "aarch64"?>
-    <?define ArchName = "arm64"?>
-    <?define ArchArchDir = "aarch64"?>
-    <?define ArchTriple = "aarch64-unknown-linux-android"?>
-    <?define ArchTripleDir = "aarch64-unknown-linux-android"?>
-  <?elseif $(ProductArchitecture) = "x86_64"?>
-    <?define ArchName = "amd64"?>
-    <?define ArchArchDir = "x86_64"?>
-    <?define ArchTriple = "x86_64-unknown-linux-android"?>
-    <?define ArchTripleDir = "x86_64-unknown-linux-android"?>
-  <?elseif $(ProductArchitecture) = "armv7"?>
-    <?define ArchName = "arm"?>
-    <?define ArchArchDir = "armv7"?>
+  <?if $(ProductArchitecture) = "arm64"?>
+    <?define Architecture = "aarch64"?>
+    <?define SwiftModuleTriple = "aarch64-unknown-linux-android"?>
+    <?define Triple = "aarch64-unknown-linux-android"?>
+  <?elseif $(ProductArchitecture) = "amd64"?>
```

Comment:
This one too. Why did the `ProductArchitecture` change from `x86_64` to `amd64`?